src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -358,4 +348,23 @@ public void testSinkTaskWithMultipleOpenClose() throws Exception {
 
     assert partitionsInTable.size() == 2;
   }
+
+  private Map<String, String> getConfig(int taskId) {
```

Comment:
could add this to testutils, but i think these methods are very customized for this test suite, so i'd rather not increase the scope?

Reply:
moved to a following pr, so this comment is out of context